src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Controllers/DashboardController.cs

```diff
@@ -43,5 +43,13 @@ public async Task<IActionResult> AssignToAssessor(Guid applicationId, int assess
 
             return RedirectToAction("ViewApplication", "Home", new { applicationId });
         }
+
+        public async Task<IActionResult> InProgressApplications()
+        {
+            var userId = _contextAccessor.HttpContext.User.UserId();
+            userId = "temp"; //TODO: Can't access the user until staff idams is enabled
+            var vm = await _orchestrator.GetInProgressApplicationsViewModel(userId);
+            return View(vm);
+        }
```

Comment:
Can you add a test for this

Reply:
Yes.  I'd set out with the intention that the controller would only have a single line to call the orchestrator but should have added the test once I started with the user details.  The test I add will have to be ignored until we can introduce staff idams tho.